Dockerfile

```diff
@@ -19,21 +19,22 @@ COPY README.md .
 COPY LICENSE .
 RUN echo 'APT::Install-Recommends "0"; APT::Install-Suggests "0";' >/etc/apt/apt.conf.d/ocr-d.conf
 RUN apt-get update && apt-get -y install software-properties-common \
-    && add-apt-repository ppa:deadsnakes/ppa \
     && apt-get update && apt-get -y install \
         ca-certificates \
-        python3.7-dev \
-        python3.7-venv \
+        python3-dev \
+        python3-pip \
+        python3-venv \
+        gcc \
```

Comment:
Oh? Why is this necessary in core now? (AFAICS we have it ocrd_all/Makefile's `deps-ubuntu` → `deps-ubuntu-modules` → `CUSTOM_DEPS`.)

Reply:
`gcc` and `python3-dev` because we need to install shapely from source in core itself, otherwise CI would fail at `make install`, wouldn't it? `python3-venv` and `python3-pip` are probably not necessary, I'll check.

Reply:
right, of course! I forgot that core needs to be self-serving. (Should perhaps get a comment though so we can remove again when not needed.)